dvc/remote/webdavs.py

```diff
@@ -0,0 +1,15 @@
+from .webdav import RemoteWEBDAV
+from dvc.scheme import Schemes
+
+
+class RemoteWEBDAVS(RemoteWEBDAV):
+    scheme = Schemes.WEBDAVS
+
+    def gc(self):
+        raise NotImplementedError
```

Comment:
do we need to repeat these stubs?

Reply:
`gc` needs to be explicitly disabled for HTTP(S) remotes since there is no way to implement `remote.remove()` for HTTP. But WebDAV does have a `DELETE` method, so should we implement `remote.remove()` for WebDAV (and by extension support `gc`)?

Reply:
@pmrowla, this works for most of the usecases already. So, I think, it's better to just not implement till any user asks for it.

Reply:
@shcheklein This code is repeating because '**DeepSource**' swears.

Reply:
👍 right. Resolving this.